third_party/build/bin/nettle_build.sh

```diff
@@ -19,7 +19,7 @@ set -e
 SCRIPT_DIR="$(dirname "$(realpath "$0")")"
```

Comment:
please add comment for every change you are making outside of docker build script.

Reply:
fixed